utils/datasets.py

```diff
@@ -0,0 +1,11 @@
+from .folders import DATASET_FOLDER
+
+TRAIN_CSV_PATH = "{}training.1600000.processed.noemoticon.csv".format(
+    DATASET_FOLDER)
+TEST_CSV_PATH = "{}testdata.manual.2009.06.14.csv".format(DATASET_FOLDER)
+
+X_TRAIN_PATH = "{}x_train.npy".format(DATASET_FOLDER)
+Y_TRAIN_PATH = "{}y_train.npy".format(DATASET_FOLDER)
+
+X_TEST_PATH = "{}x_test.npy".format(DATASET_FOLDER)
+Y_TEST_PATH = "{}y_test.npy".format(DATASET_FOLDER)
```

Comment:
A bit strange to have `folders.py` containing paths to directories and no `files.py` that would contain paths to interesting files but it's a detail

Reply:
Fair comment. I did that because we have general folders with no links between them. But concerning the files, they all share the same purpose : the dataset. So I found it more appropriate to put them in `datasets.py`.